xml/System.Data.Common/DbDataReader.xml

```diff
@@ -2154,7 +2097,7 @@ private static void GetCredits(String connectionString) {
         <AssemblyVersion>4.2.0.0</AssemblyVersion>
       </AssemblyInfo>
       <Docs>
-        <summary>An asynchronous version of <see cref="M:System.Data.Common.DbDataReader.IsDBNull(System.Int32)" />, which gets a value that indicates whether the column contains non-existent or missing values.</summary>
+        <summary>Asynchronously gets a value that indicates whether the column contains non-existent or missing values.</summary>
```

Comment:
This is a boolean value. Is there a better pattern for this?

Reply:
I'm not sure I understand the question...

Another thought is to include a note on this (and on GetFieldValueAsync) that clarifies that these typically only make sense if SequentialAccess is specified.

Reply:
I refer to a better pattern for how to document a method that returns a bool value that indicates something. For example, I was thinking to remove the whole "Get a value" and just say "indicates whether the column contains...". 

I agree on the SequentialAccess note.